ingest/queries/describeProject.rq

```diff
@@ -0,0 +1,56 @@
+PREFIX rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs:  <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX xsd:   <http://www.w3.org/2001/XMLSchema#>
+PREFIX owl:   <http://www.w3.org/2002/07/owl#>
+PREFIX dco: <http://info.deepcarbon.net/schema#>
+PREFIX dc: <http://purl.org/dc/elements/1.1/>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX vivo: <http://vivoweb.org/ontology/core#>
+PREFIX vitro: <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#>
+PREFIX vitro-pub: <http://vitro.mannlib.cornell.edu/ns/vitro/public#>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+PREFIX bibo: <http://purl.org/ontology/bibo/>
+PREFIX obo: <http://purl.obolibrary.org/obo/>
+
+DESCRIBE ?project ?dcoid ?mostSpecificType ?mainImage ?thumbImage ?thumbnail ?description ?leader ?community ?portalGroup ?submittedBy ?locationLabel ?role ?participant ?contributingRole ?contributor ?dateTimeInterval ?grant ?start ?end ?startDateTime ?endDateTime ?reportingYear ?pu
+WHERE {
+  ?project dco:hasDcoId ?dcoid .
+  ?project vitro:mostSpecificType ?mostSpecificType .
+  ?project vivo:description ?description .
+  OPTIONAL { ?project vivo:hasFundingVehicle ?grant }
+  OPTIONAL { ?project dco:fieldworkLeader ?leader }
+  OPTIONAL { ?project dco:associatedDCOCommunity ?community }
+  OPTIONAL { ?project dco:associatedDCOPortalGroup ?portalGroup }
+  OPTIONAL { ?project dco:submittedBy ?submittedBy }
+  OPTIONAL { ?project dco:hasProjectUpdate ?pu .
+    OPTIONAL{ ?pu dco:forReportingYear ?reportingYear }
```

Comment:
I don't think this needs to be optional (forReportingYear). We're looking for the reporting year, so the project update either has one or it doesn't